docs/quick_start/gui.rst

```diff
@@ -1,11 +1,31 @@
 .. Images
+.. |login| image:: ../images/gui_login.jpg
```

Comment:
Please review the screenshot again; there are highlights to the right of the input boxes that might confuse users

Reply:
That is how it always looks on my browser (Chrome). It could be a browser related difference or related to having LastPass installed.

Reply:
Hmm with password manager